BeatTogether.DedicatedServer.Kernel/DedicatedInstance.cs

```diff
@@ -401,33 +402,34 @@ public override void OnDisconnect(EndPoint endPoint, DisconnectReason reason)
             {
                 // Set new manager if manager left
                 if (Configuration.ManagerId == "" && Configuration.GameplayServerMode == Enums.GameplayServerMode.Managed)
+                {
```

Comment:
I put the code for getting the new manager all into the if statement instead of just line 406.
Fixes crash where when a player leaves quickplay, the server crashes because it cannot find the user with the userid "ziuMSceapEuNN7wRGQXrZg" because that user does not exist. fixes issue https://github.com/BeatTogether/BeatTogether.DedicatedServer/issues/11#issue-1174473474